internal/users/follow_unfollow.go

```diff
@@ -0,0 +1,75 @@
+package users
+
+import (
+	"Twitter_like_application/internal/database/pg"
+	"Twitter_like_application/internal/services"
+	"encoding/json"
+	"fmt"
+	"net/http"
+)
+
+func FollowUser(w http.ResponseWriter, r *http.Request) {
+	userID := r.Context().Value("userID").(int)
+
+	var targetUserID TargetUser
+	err := json.NewDecoder(r.Body).Decode(&targetUserID)
+	if err != nil {
+		services.ReturnErr(w, err.Error(), http.StatusBadRequest)
+		return
+	}
+
+	var count int
+	err = pg.DB.QueryRow("SELECT COUNT(*) FROM followers_subscriptions WHERE follower_id = $1 AND subscription_id = $2", userID, targetUserID.ID).Scan(&count)
```

Comment:
you should not use `COUNT(*)` to check existence of something, please use `EXISTS` instead, or `SELECT 1 ... LIMIT 1`